src/style.css

```
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  padding: 0 8px;
+}
+
+.search__bar {
+  margin-top: 20px;
+  display: flex;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search__bar--big {
+  height: 70px;
+}
+
+.search__bar--small {
+  height: 42px;
+}
+
+.search__bar-label {
+  display: flex;
+  align-items: center;
+  width: 100%;
+}
+
+.search__bar-img {
+  margin: 0 17px 0 26px;
+}
+
+.search__bar-img--small {
+  width: 11px;
+  height: 11px;
+  margin: 0 10px 1px 12px;
+}
+
+.search__bar-input {
+  width: 100%;
+  border-radius: 4px;
+  border: none;
+}
+
+.search__bar-input--big {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search__bar-input--small {
+  font-family: "Avenir", sans-serif;
+  font-size: 14px;
+}
```

font-family is already setted for html, for input use inherit value. Also do not repeat ypur self
```suggestion
.search__bar-input {
  width: 100%;
  border-radius: 4px;
  border: none;
  font-family: inherit;
}

.search__bar-input--big {
  font-size: 16px;
}

.search__bar-input--small {
  font-size: 14px;
}
```